settings.gradle

```diff
@@ -24,9 +24,11 @@ def includeSubprojectPerVersion(flavor,classification) {
                 def version = projectFile.name
                 def projectName = "upstream:${flavor}-${version}"
 
-                include projectName
-                project(":${projectName}").projectDir = file("upstream/${flavor}/${version}")
-                println "included $projectName"
+                if (new File("$projectFile/overload.gradle").exists()) {
```

Comment:
Why do we need to check if this overload exists everywhere?

Reply:
This is there to catch directories that are left behind when a subproject is renamed.

Reply:
So same comment as on https://github.com/xenit-eu/docker-alfresco/pull/69: please add a comment to the code.

Reply:
Would have been faster if you added it yourself.